src/components/agencyspecific.ts

```diff
@@ -0,0 +1,148 @@
+            '399': 'SPRINTER',
+        }
+    }
+
+    // @ts-ignore
+    if (fixPatterns[agency]) {
+        // @ts-ignore
+        return fixPatterns[agency][rid] || fixPatterns[agency]['*'] || route.replace('Counterclockwise', 'Anticlockwise')
+    } else {
+        return route
+    }
+}
+
+export function fixRouteNameLong(agency: string, route: string, rid: string): string {
+    let fixPatterns = {
+        'metrolinktrains': {
+            'Metrolink 91/Perris Valley Line': '91/Perris Valley Line',
+            'Metrolink Antelope Valley Line': 'Antelope Valley Line',
+            'Metrolink Inland Empire-Orange County Line': 'Inland Empire-OC Line',
+            'Metrolink Orange County Line': 'Orange County Line',
+            'Metrolink Riverside Line': 'Riverside Line',
+            'Metrolink San Bernardino Line': 'San Bernardino Line',
+            'Metrolink Ventura County Line': 'Ventura County Line',
+        },
+        'san-diego-mts': {
+            '3': 'Ocean View Boulevard/Hillcrest',
+            '5': 'Market Street',
+            '215': 'Mid-City Rapid',
+            '225': 'South Bay Rapid',
+            '235': 'I-15 Rapid',
+            '201': 'SuperLoop',
+            '202': 'SuperLoop',
+            '204': 'SuperLoop',
+            '227': 'Iris Rapid',
+            '237': 'Mira Mesa Rapid',
+            '280': 'Rapid Express',
+            '290': 'Rapid Express',
+            '398': 'COASTER',
+            '399': 'SPRINTER',
+            'AIR': 'Flyer'
+        }
+    }
+
+    // @ts-ignore
+    if (fixPatterns[agency]) {
+        // @ts-ignore
+        return fixPatterns[agency][rid] || fixPatterns[agency]['*'] || route.replace('Counterclockwise', 'Anticlockwise')
+    } else {
+        return route.replace('Transit Station', 'Sta').replace('Station', 'Sta').replace('Transportation Center', 'TC').replace('Transit Center', 'TC').replace('Transit Ctr', 'TC').trim()
+    }
+}
+
+export function fixRunNumber(agency: string, type: number, route: number, tripname: string, vehicle: string): string | null {
+    if (agency == 'san-diego-mts' && type == 0) return vehicle
+    if (agency == 'northcountytransitdistrict' && route != 398) return null
+    return tripname
+}
+
+export function fixStationName(name: string) {
+    console.log(name)
+    let fixPatterns = [
+        ['L.A. Union Station', 'Los Angeles'],
+        ['North Hollywood Station', 'NoHo'],
+        ['North Hollywood Station G - Line', 'NoHo'],
+        ['Chatsworth Station G - Line', 'Chatsworth'],
+        ['Union Station', 'UnionSta'],
+        ['Wilshire / Western Station', 'Wil/Wstrn'],
+        ['Wilshire / Vermont Station', 'Wilshire Ctr'],
+        ['Downtown Long Beach Station', 'Long Bch'],
+        ['APU / Citrus College Station', 'Azusa'],
+        ['Redondo Beach Station', 'Redondo Bch'],
+        ['Norwalk Station', 'Norwalk'],
+        ['Downtown Santa Monica Station', 'S Monica'],
+        ['Atlantic Station', 'East LA'],
```

Comment:
Why are we changing Atlantic Station to East LA? This should remain the same.

Reply:
Abbreviations were copied from sign mode, cleaning them up now, especially since LA Metro provides no headsign data